packages/api/src/index.ts

```
@@ -1,15 +1,27 @@
 import express, { Express, Request, Response, NextFunction } from 'express';
-import { HealthStatus, ApiResponse, API_ENDPOINTS, PathRequest, RouteResponse, RouteCalculationResponse, RoutePathNode, RouteSegment } from '@campus-nav/shared/types';
+import { HealthStatus, ApiResponse, API_ENDPOINTS, RouteResponse, RouteCalculationResponse, RoutePathNode, RouteSegment } from '@campus-nav/shared/types';
```

ESLint

[error] 'HealthStatus' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)